torch/csrc/distributed/rpc/rref_context.h

```diff
@@ -105,7 +102,7 @@ class TORCH_API RRefContext {
   // available, e.g., when processing to_here(). The forceCreated flag can be
   // used to ensure that the rref is created on the owner, otherwise throw in
   // cases where the user of this API expects this to return a completed future.
-  std::shared_ptr<Future<c10::intrusive_ptr<OwnerRRef>>> getOwnerRRef(
+  c10::intrusive_ptr<JitFuture> getOwnerRRef(
```

Comment:
Since `JitFuture` does not explicitly name a type in the signature, shall we add a comment to explain what's in the JiFuture?